ui/src/components/ApprovalsTracker/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+import * as Types from '@types';
+import * as Interfaces from '@interfaces';
+
+type Props = {
+    publicationData: Interfaces.Publication;
+    user: Types.UserType;
+};
+
+const ApprovalsTracker: React.FC<Props> = (props): React.ReactElement => {
+    return (
+        <>
+            <p className="my-2 text-lg dark:text-white-50">
+                Your role on this publication:{' '}
+                {props.user.id === props.publicationData.createdBy ? 'Corresponding author' : 'Author'}
+            </p>
+            <div className="my-2 overflow-hidden rounded-lg shadow ring-1 ring-black ring-opacity-5 dark:ring-transparent">
+                <table
```

Comment:
There should be more space between the table and the publication title

Reply:
On mobile devices, if the email address is long enough, there's no way to see 'Status' column
![image](https://user-images.githubusercontent.com/48569671/218036872-f6efb071-9ffb-40d2-b4a2-b741e7edc4cb.png)

Horizontal scroll should be available if the content overflows